allure-commandline/src/main/java/ru/yandex/qatools/allure/logging/Message.java

```diff
@@ -1,37 +0,0 @@
-package ru.yandex.qatools.allure.logging;
-
-import ch.qos.cal10n.BaseName;
-import ch.qos.cal10n.Locale;
-import ch.qos.cal10n.LocaleData;
-
-/**
- * @author Artem Eroshenko <eroshenkoam@yandex-team.ru>
- */
-@BaseName("message")
-@LocaleData(value = {@Locale("en"), @Locale("ru")}, defaultCharset = "UTF-8")
-public enum Message {
```

Comment:
Airlifts airline doesn't support localization at all. And I don't like partially localized tools